### PR TITLE
 4 finish bottom navigation styling

### PR DESCRIPTION
**Pull request by aniamusial**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/4

# Scope of work
Finished styling of bottom navigation icons. Separate slots for image icon and floating button - this element should be implemented after (https://github.com/DivanteLtd/storefront-ui/issues/25)

- In my opinion, there's no need to split into smaller atoms. 

# Screenshots of visual changes
Screenshot from 2019-03-27 15-22-51

### CODE REVIEW

**Comment by filrak** [comment]
2nd icon is different on designs

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.js [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+      default: false
+    }
+  },
+  computed: {
+    currentIcon: {
+      get() {
+        return this.isActive;
+      },
+      set(value) {
+        this.$emit("change:isActive", value);
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.js]: change to `current`

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.js [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+      }
+    }
+  },
+  methods: {
+    setCurrentIcon: value => {
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.js]: It doesn't need to be icon. `setCurrent` is better ;)

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# SfBottomNavigation
+
+Mobile-only bottom navigation menu with icon elements and floating Button.
+
+## HTML Template
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.md]: Remove html, now we use different template (hTML is now automatically added to info panel) + storybook can't parse it in this form ;/

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.scss [hunk 4]
```diff
@@ -1,37 +1,52 @@
-@import '../../../css/variables';
+@import "../../../css/variables";
+
+$bottom-navigation-cart-shadow: 0px 4px 6px rgba(66, 66, 66, 0.2) !default;
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.scss]: it should be `__cart` and not curt but just `__floating-btn`

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.spec.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { shallowMount } from "@vue/test-utils";
+import SfBottomNavigation from "@/components/organisms/SfBottomNavigation/SfBottomNavigation.vue";
+
+describe("SfBottomNavigation.vue", () => {
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.spec.ts]: add tests

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.stories.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text } from "@storybook/addon-knobs";
+import SfBottomNavigation from "./SfBottomNavigation.vue";
+
+export default storiesOf("BottomNavigation", module)
+  .addDecorator(withKnobs)
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.stories.js]: adjust stories to common format (check banner) 
- props
- scss mods
- one for every prop

also include  markdown notes (search for `notes` in other stories)

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+      </Sf-Circle-Button>
+      </slot>
+  </div>
+  <slot />
+</div>
+```
+
+## Slots
+
+- `icon` - customizable icon icon, intended for navigation menus
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.md]: icon icon? ;d

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
+# SfBottomNavigation
+
+Mobile-only bottom navigation menu with icon elements and floating Button.
+
+## HTML Template
+
+```html
+<div class="sf-bottom-navigation">
+  <div class="sf-bottom-navigation__elements" v-for="icon in icons">
+    <slot name="icon" :image="icon">
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.md]: change slot name to `items`

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+          :src="icon"
+          alt="nav-home"
+          class="sf-bottom-navigation-element-img"
+        />
+      </div>
+    </slot>
+  </div>
+<div class="sf-bottom-navigation__element">
+      <Sf-Circle-Button class="sf-bottom-navigation-cart-button">
+        <img src="/assets/cart.svg" alt="nav-cart" class="sf-bottom-navigation-element-img"/>
```
Comment (filrak) [inline src/components/organisms/SfBottomNavigation/SfBottomNavigation.md]: make this icon customizable via prop and slot (both `floating-icon`)
and make this buttonj optional (exactly like SfButton in in SfBanner

File: src/components/organisms/SfBottomNavigation/SfBottomNavigation.scss (hunk 4 above)
Comment (aniamusial) [reply src/components/organisms/SfBottomNavigation/SfBottomNavigation.scss]: I've deleted it, it's in SfCircleButton now.

**Pull request closed without merging**